django/contrib/admin/templates/registration/password_reset_form.html

```diff
@@ -16,7 +17,13 @@
 
 <form method="post">{% csrf_token %}
 {{ form.email.errors }}
```

Comment:
In the other template, the errors are in the <div> but here they are outside, should it be consistent?